example/install_rackhd.sh

```diff
@@ -0,0 +1,252 @@
+#
+#############################################
+
+install_prerequisite(){
+
+    sudo apt-get -y install rabbitmq-server
+    sudo apt-get -y install mongodb
+    sudo apt-get -y install snmp
+    sudo apt-get -y install ipmitool
+
+    sudo apt-get -y install ansible
+    sudo apt-get -y install apt-mirror
+    sudo apt-get -y install amtterm
+
+    sudo apt-get -y install isc-dhcp-server
+
+}
+
+#############################################
+#  Install RackHD deb package from bintray
+#
+#############################################
+install_RackHD(){
+    echo "deb https://dl.bintray.com/rackhd/debian trusty main" | sudo tee -a /etc/apt/sources.list
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
+    sudo apt-get update
+
+   echo "[INFO] Install RackHD ........"
+
+    sudo apt-get -y install on-dhcp-proxy on-http on-taskgraph
+    sudo apt-get -y install on-tftp on-syslog
+}
+
+#############################################
+#  Update DHCP config
+#
+#############################################
+update_dhcp_config(){
+
+    # After isc-dhcp-server installed, there should be /etc/dhcp/dhcpd.conf file
+    sudo echo "
+    ##### RackHD added lines########################
+    deny duplicates;
+    ignore-client-uids true;
+    subnet 172.31.128.0 netmask 255.255.252.0 {
+       range 172.31.128.2 172.31.131.254;
+       # Use this option to signal to the PXE client that we are doing proxy DHCP
+       option vendor-class-identifier \"PXEClient\";
+    }
+    " >> /etc/dhcp/dhcpd.conf
+}
+#############################################
+#  Do tricky job to entertain upstart(Only for Ubuntu 14.04)
+#
+#############################################
+make_upstart_happy(){
+    #
+    #  The services files in /etc/init/ all need a conf file to exist in /etc/default/{service} Touch those files to allow the upstart scripts to start automatically.
+    #
+    for service in $(echo "on-dhcp-proxy on-http on-tftp on-syslog on-taskgraph");
+    do touch /etc/default/$service;
+    done
+}
+
+#############################################
+# Copy example config.json , and put it into /opt/monorail/config.json
+#
+# NOTE: this is only for quick tutorial, modify the config for your own situation
+#############################################
+create_RackHD_config(){
+    wget https://raw.githubusercontent.com/RackHD/RackHD/master/packer/ansible/roles/monorail/files/config.json  -O /opt/monorail/config.json
+}
+#############################################
+# Copy the PXE image and micro kernel from bintray 
+#
+#############################################
+copy_RackHD_static_bins(){
+
+    echo "[INFO] Will Copy static files, it will take a while"
+
+    mkdir -p /var/renasar/on-tftp/static/tftp
+    cd /var/renasar/on-tftp/static/tftp
+
+    for file in $(echo "\
+        monorail.ipxe \
+        monorail-undionly.kpxe \
+        monorail-efi64-snponly.efi \
+        monorail-efi32-snponly.efi");do
```

Comment:
missing the `monorail.intel.ipxe` - only needed for NUC installation - per recent PR: https://github.com/RackHD/on-imagebuilder/pull/51